generator.rb

```diff
@@ -0,0 +1,112 @@
+def is_vowel?(letter)
+  vowel_array = ["a", "e", "i", "o", "u"]
+  if vowel_array.include? letter.downcase
+    # if letter == "a" || letter == "e" || letter == "i" || letter == "o" || letter == "u"
+    return true
+  else
+    return false
+  end
+end
+
+def run_generator
+  print "Please enter the first word: "
+  wordA = gets.chomp
+
+  #make sure each input for wordA and wordB is two or more characters
+  until wordA.length > 1
+    print "Please make sure your word is at least two characters long. Try again: "
+    wordA = gets.chomp
+  end
+
+  print "Please enter the second word: "
+  wordB = gets.chomp
+
+  until wordB.length > 1
+    print "Please make sure your word is at least two characters long. Try again: "
+    wordB = gets.chomp
+  end
+
+  #wordA: iterate through word backwards and find the index for the first vowel
+  wordA_index = wordA.length - 1
+  vowels_tracker = 0
+  ## tried attaching downcase method to line 54 but didn't allow .each_char method
+  # wordA = wordA.downcase || use if you want output to be lowercased
+  wordA.reverse.each_char do |letter|
+    if is_vowel?(letter) == false && vowels_tracker < 1
+      wordA_index -= 1
+    else
+      vowels_tracker += 1
+    end
+  end
```

Comment:
I like your strategy here! Actually, I think that `vowels_tracker` is very useful and interesting, but I was wondering about the value you give it... it starts at `0`. What does it represent? What do you use it for? You end up not using the number of `vowels_tracker` for anything in the rest of the code. You use it in the conditional for `vowels_tracker < 1`. In this case, it might be better if `vowels_tracker` was a boolean that represented if the first vowel had been found.